sympy/polys/domains/mpelements.py

```diff
@@ -45,7 +45,7 @@ def parent(self):
 @public
 class MPContext(PythonMPContext):
 
-    def __init__(ctx, prec=53, dps=None, tol=None):
+    def __init__(ctx, field, prec=53, dps=None, tol=None):
```

Comment:
I would use a flag with default value like `real=False`. It should also come after the existing parameters so that calls with positional parameters would not be disturbed.